src/main/java/net/neoforged/neoforge/event/level/BlockEvent.java

```diff
@@ -99,6 +99,49 @@ public void setExpToDrop(int exp) {
         }
     }
 
+    /**
+     * Fired when a block is destroyed AFTER drops have been determined, but have not yet spawned.
+     * - Cancellation will result in the drops not being spawned.
+     * - Breaker may be null if the block was not destroyed by an entity.
+     * - The drops list is immutable and cannot be modified. Use LootTableModifiers instead
```

Comment:
This information should be in the method-level javadoc, not the class javadoc.

Reply:
Javadoc has been reworked.